docs/demo/build.md

```diff
@@ -4,12 +4,12 @@
 - Go 1.13
 
 ## Developer Setup
-In order to access edge service image and user agent image you have to create personal token with read:packages and repo permissions ([personal token](https://help.github.com/en/github/authenticating-to-github/creating-a-personal-access-token-for-the-command-line)).
+In order to access edge service image and user agent image you have login to docker.pkg.github.com.
  
-Run the following command using your newly generated personal token:
+Run the following command using your github username:
 
 ```
-docker login -u <username> -p <github token with read:packages permission> docker.pkg.github.com
+docker login -u <username> -p e438ec2cd264600e6d3fec6dd8aeda1633e4f929 docker.pkg.github.com
```

Comment:
I'm not sure I understand this change. You need to create a GitHub token with these permissions.